el/index.js

```diff
@@ -93,6 +106,26 @@ module.exports = yeoman.generators.Base.extend({
       this.writeFileFromString(file, 'app/elements/elements.html');
     }
 
+    if (this.testType && this.testType !== 'None') {
+      var testDir = 'app/test';	
+      
+      if (this.testType === 'TDD') {
+        this.template(path.join(__dirname, 'templates/test/_tdd.html'), path.join(testDir, this.elementName+'-basic.html'));
+      } else if (this.testType === 'BDD') {
+        this.template(path.join(__dirname, 'templates/test/_bdd.html'), path.join(testDir, this.elementName+'-basic.html'));
+      }
+
+      // Open index.html, locate where to insert text, insert ", x-foo.html" into the array of components to test
+      var indexFileName = 'app/test/index.html';
+      var origionalFile = this.readFileAsString(indexFileName);
+      var regex = /WCT\.loadSuites\(\[([^\]]*)/;
+      var match = regex.exec(origionalFile);
+      var indexOfInsertion = match.index + match[0].length;
+      var comma = (match[1].length === 0) ? '' : ', ';
+      var newFile = origionalFile.slice(0, indexOfInsertion) + comma + '\'' + this.elementName + '-basic.html\'' + origionalFile.slice(indexOfInsertion);
```

Comment:
Yeah I think I meant cleaner than having to edit the string and manually insert commas and stuff like that. And as you said, it may be easier for someone glancing at the code to grok what's going on